deps/publishing/src/logseq/publishing.cljs

```diff
@@ -1,4 +1,4 @@
-(ns logseq.publishing
+(ns ^:node-only logseq.publishing
```

Comment:
I've started using a :node-only metadata tag to be explicit about when a ns can only be used by electron and nbb-logseq